templates/default/rt/rtticketaddscripts.html

```diff
@@ -0,0 +1,76 @@
+<SCRIPT type="text/javascript">
```

Comment:
to by było fajnie wydzielić do pliku .js w katalogu img, czyli takiego, który w ogóle nie ma nawiązań do smarty.